README.md

```diff
@@ -93,5 +95,14 @@
 * https://play.google.com/about/developer-content-policy/
 * https://developer.android.com/distribute/best-practices/launch/launch-checklist
 
+#### Animation 
```

Comment:
Although animations are nice, it is not in the roadmap. Maybe you should open an issue regarding the inclusion of this topic in the roadmap instead?